core/src/main/java/org/frankframework/http/HttpSender.java

```diff
@@ -141,51 +140,26 @@ protected HttpRequestBase getMethod(URI url, Message message, ParameterValueList
 			}
 		}
 
-		URI uri;
-		try {
-			uri = encodeQueryParameters(url);
-		} catch (UnsupportedEncodingException | URISyntaxException e) {
-			throw new SenderException("error encoding queryparameters in url ["+url.toString()+"]", e);
-		}
-
-		if(postType==PostType.URLENCODED || postType==PostType.FORMDATA || postType==PostType.MTOM) {
+		if (postType == PostType.URLENCODED || postType == PostType.FORMDATA || postType == PostType.MTOM) {
 			try {
-				return getMultipartPostMethodWithParamsInBody(uri, message, parameters, session);
+				return getMultipartPostMethodWithParamsInBody(url, message, parameters, session);
 			} catch (IOException e) {
 				throw new SenderException("unable to read message", e);
 			}
 		}
 		// RAW + BINARY
-		return getMethod(uri, message, parameters);
-	}
-
-	// Encode query parameter values.
-	private URI encodeQueryParameters(URI url) throws UnsupportedEncodingException, URISyntaxException {
```

Comment:
The result of this method was always URLs with special characters double-encoded.

The only reason we didn't have more errors, is that later we decoded them again and then added the decoded params back as if already encoded.